src/components/BrowseSkill/BrowseSkill.js

```diff
@@ -459,21 +455,22 @@ export default class BrowseSkill extends React.Component {
           topGames: data.metrics['Games, Trivia and Accessories'],
         });
       },
-      error: function(e) {
+      error: e => {
         console.log('Error while fetching skills based on top metrics', e);
-        return self.loadMetricsSkills();
+        return this.loadMetricsSkills();
       },
     });
   };
 
   handleRatingRefine = ratingRefine => {
-    let prevRatingRefine = this.state.ratingRefine;
+    const { skills } = this.state;
```

Comment:
Deconstruct ratingRefine here

Reply:
ratingRefine is also the argument of the function, so we are getting a declaration conflict. That is why I had not used deconstruct for rating refine. @akshatnitd

Reply:
@akshatnitd It would end up clashing with the ratingRefine of the argument.